### ISSUE TITLE
Translation metadata format

### ISSUE THREAD

**Issue by LoicGrobol** [UniversalDependencies/UD_Breton-KEB#22]
Currently the translations are formatted this way

```conllu
# text = N'int ket aet war-raok.
# text[eng] = They didn't progress.
# text[fra] = Ils n'ont pas progressé.
```

I don't believe it has been standardized, but [the doc](https://universaldependencies.org/format.html#sentence-boundaries-and-comments) would suggest to rather format the metadata keys this way

```conllu
# text = N'int ket aet war-raok.
# text_en = They didn't progress.
# text_fr = Ils n'ont pas progressé.
```

**Comment by ftyers** [comment]
It hasn't breen standardised, but all the treebanks I have produced are done like that. Using ISO-639-3 codes and square brackets.

**Comment by dan-zeman** [comment]
> but [the doc](https://universaldependencies.org/format.html#sentence-boundaries-and-comments) would suggest

It was put in the documentation simply to increase chances that it will be done similarly in multiple treebanks. All the treebanks I have produced are done like that :-) But it is not a guideline approved by the core group or checked by the validator.

**Comment by bguil** [comment]
Indeed, there is a zoo of 357 different metadata used in 2.11 treebanks: see http://tables.grew.fr/?data=meta/META.
